lib/WooCommerce/TransactionalEmails.php

```diff
@@ -94,12 +100,12 @@ public function useTemplateForWoocommerceEmails() {
 
   private function createNewsletter() {
     $wc_email_settings = $this->getWCEmailSettings();
-    return Newsletter::createOrUpdate([
-      'type' => Newsletter::TYPE_WC_TRANSACTIONAL_EMAIL,
-      'subject' => 'WooCommerce Transactional Email',
-      'preheader' => '',
-      'body' => json_encode($this->template->create($wc_email_settings)),
-    ]);
+    $newsletter = new NewsletterEntity;
+    $newsletter->setType(NewsletterEntity::TYPE_WC_TRANSACTIONAL_EMAIL);
+    $newsletter->setSubject('WooCommerce Transactional Email');
+    $newsletter->setBody(json_encode($this->template->create($wc_email_settings)));
```

Comment:
You don't have to run `json_encode` here, the type on the entity is JSON. I wonder how does that even work.